From 70daf1003996f5f1c30db0406310483b4f389c4e Mon Sep 17 00:00:00 2001
From: Joseph Chen <chenjh@rock-chips.com>
Date: Wed, 22 Nov 2017 15:14:56 +0800
Subject: [PATCH] ARM: dts: rk3036-kylin: set vdd_arm regulator-on-in-suspend

rk3036 doesn't support vdd_arm power supply off when system suspend.

Change-Id: I46bd8a7c2b672be30d8106b867275e8ba7d77e54
Signed-off-by: Joseph Chen <chenjh@rock-chips.com>
---
 arch/arm/boot/dts/rk3036-kylin.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk3036-kylin.dts b/arch/arm/boot/dts/rk3036-kylin.dts
index ac4bd5b48ae0..1d1f991538ef 100644
--- a/arch/arm/boot/dts/rk3036-kylin.dts
+++ b/arch/arm/boot/dts/rk3036-kylin.dts
@@ -229,7 +229,8 @@
 				regulator-max-microvolt = <1350000>;
 				regulator-name = "vdd_arm";
 				regulator-state-mem {
-					regulator-off-in-suspend;
+					regulator-on-in-suspend;
+					regulator-suspend-microvolt = <1050000>;
 				};
 			};
 
-- 
2.35.3

